Automattic/wp-calypso
# composite-checkout: Replace styled-components with emotion

#### Changes proposed in this Pull Request

Mirroring the decision in https://github.com/WordPress/gutenberg/pull/17963, we're going to swap out `styled-components` with emotion.

#### Testing instructions

- Add a file called packages/composite-checkout/demo/private.js and inside it put the following: `export const stripeKey = 'TEST KEY HERE'`;
- Run `npm run composite-checkout-demo`.
- Verify that the form works as expected.
